openmole/gui/server/org.openmole.gui.server.core/src/main/scala/org/openmole/gui/server/core/Utils.scala

```diff
@@ -181,11 +181,11 @@ object Utils extends JavaLogger {
     val toPath = safePath.copy(path = safePath.path.dropRight(1) :+ newName)
     if (toPath.isDirectory()) toPath.mkdir
 
-    val parent = safePath.parent
-    safePath.copy(safePath.parent, Some(newName))
+    val from: File = safePath
+    val replica: File = safePath.parent ++ newName
+    replica.content = from.content
```

Comment:
This is very problematic for 2 reasons:
 * it doesn't work with directory
 * it load the entire file in the memory (inefficient and may crash the node with out of mem in case of big files)